src/appleseed.shaders/src/appleseed/as_globals.osl

```diff
@@ -136,33 +136,73 @@ shader as_globals
         string help = "Partial derivative of P along the Z direction.",
         int divider = 1
     ]],
-    output normal out_world_dNdu = 0
+    normal dNdu = 0
     [[
         int lockgeom = 0,
+        string widget = "null",
+        int as_maya_attribute_connectable = 0,
+        int as_maya_attribute_keyable = 0,
+        int as_maya_attribute_hidden = 1,
+        int as_blender_input_socket = 0,
+        int gafferNoduleLayoutVisible = 0,
+        string widget = "null"
```

Comment:
You will need to rebase on top of master and add `as_max_param_id` values to all new attributes **in shaders that already use them** (NOT the other shaders).